doc/ref_impl/cntt-ri/chapters/chapter06.md

```diff
@@ -61,241 +63,400 @@ Thanks to the descriptor file, the NFVi infrastructure deployment could be compl
+| HW_REGION | String | Yes | Hardware region |
+| MODULE_NAME | String | Yes | Hardware module which is devided by location, like area A module 1 |
+
+<p align="center"><b>Table 6-3-5:</b> Network device information.</p>
 
-<p align="center"><b>Table 6-3-5:</b> Network information.</p>
 
 <a name="6.3.6"></a>
-### 6.3.6 Controller information
-List of controller nodes that designed for current VIM deployment.
+### 6.3.6 Port mapping information
+Wiremap defines the port mapping between server/switch and switch for each line, 
+we will need this information to trace the connected server and port, so we can extrapolate the required network configuration for the port.
+
+Multiple instances are expected, one instance for each physical cable.
 
 | Field # | type | mandatory | Instruction |
 |----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| device_name | String | Yes | must be the device_name defined in server_information(###6.3.3), which would be set on Server from BMC|
-| node_name | String | Yes | node name for the controller to deploy, e.g: controller01 |
+| NAME | String | Yes | Name of network device |
+| LOCAL_RACK | String | Yes | the rack info for local device   |
+| LOCAL_NAME | String | Yes | local device name, LOCAL_NAME must reference either "Network Device Name" from table 6.3.5 |
+| LOCAL_TYPE | String | Yes | Local device type, switch or server |
+| LOCAL_PORT | String | Yes | connected port in local device |
+| REMOTE_RACK | String | Yes |  |
+| REMOTE_NAME | String | Yes | connected remote device name, REMOTE_NAME must reference either "Network Device Name" from table 6.3.5 or "Server Name" from table 6.3.4.1 |
+| REMOTE_TYPE | String | Yes | remote device type, it can be switch or server |
+| REMOTE_PORT | String | Yes | connected port in remote device. When describing port for remote servers, we use port number like 1_1, or 1_2, instead of PCIeslot number, because the server NIC mapping is already defined in 6.3.4.2 |
+| LINE_TYPE | String | Yes | Line type to describe local device type and remote device type, how each line is connected. For example "S-SRV-C_S-TOR" means this line is connecting a service server in compute module to service TOR, and another example "ST-SRV-S_M-TOR" means storage server connecting to a manage TOR in storage module. The line type can be customized defined, as long as it's unified in end user.|
+
+<p align="center"><b>Table 6-3-6:</b> Port mapping information.</p>
 
-<p align="center"><b>Table 6-3-6:</b> Controller information.</p>
 
 <a name="6.3.7"></a>
-### 6.3.7 Compute information
-List of compute nodes that designed for current VIM deployment.
+### 6.3.7 Network planning information
+Network planning info for resource pool needs to be defined, which should include Vlan ID, allocated IP range, the applied node set.
```

Comment:
Network planning information for the resource pool of each node needs to be defined which should include VLAN ID an allocated IP range.

Reply:
Thanks! updated